doc/tutorial/index.rst

```diff
@@ -0,0 +1,198 @@
+
+This will present you a series of questions
+required to create the basic directory and configuration layout for your project
+inside the `docs/` folder.
+To proceed, introduce these answers:
+
+- ``> Separate source and build directories (y/n) [n]``: Write "``y``" (without quotes)
+  and press :kbd:`Enter`.
+- ``> Project name``: Write "``Lumache``" (without quotes)
+  and press :kbd:`Enter`.
+- ``> Author name(s)``: Write "``Graziella``" (without quotes)
+  and press :kbd:`Enter`.
+- ``> Project release []``: Write "``0.1``" (without quotes)
+  and press :kbd:`Enter`.
+- ``> Project language [en]``: Leave it empty (the default, English)
+  and press :kbd:`Enter`.
+
+After the last question,
+you will see the new ``docs`` directory with some content::
+
+	docs/
+	├── build
+	├── make.bat
+	├── Makefile
+	└── source
+		├── conf.py
+		├── index.rst
+		├── _static
+		└── _templates
+
+These files are:
+
+- ``build/``: An empty directory (for now)
+  that will hold the rendered documentation.
+- ``make.bat`` and ``Makefile``: Convenience scripts
+  to simplify some common Sphinx operations,
+  such as rendering the content.
+- ``source/conf.py``: A Python script holding the configuration of the Sphinx project.
+  It contains the project name and release you specified to ``sphinx-quickstart``,
+  as well as some extra configuration keys.
+- ``source/index.rst``: The :term:`master document` of the project,
+  which serves as welcome page
+  and contains the root of the "table of contents tree" (or *toctree*).
+
+Thanks to this bootstrapping step,
+you already have everything needed
+to render the documentation as HTML for the first time.
+To do that, run this command:
+
+.. code-block:: bash
+
+   (.venv) $ sphinx-build -b html docs/source/ docs/build/html
+
+And finally, open `docs/build/html/index.html` in your browser.
+You should see something like this:
+
+.. image:: /_static/tutorial/lumache-first-light.png
+
+There we go! You created your first HTML documentation using Sphinx.
+
+Making some tweaks to the index
+...............................
+
+The ``index.rst`` file that ``sphinx-quickstart`` created
+has some content already,
+and it gets rendered as the front page of our HTML documentation.
+It is written in reStructuredText,
+a powerful markup language.
+
+Modify the file like follows:
+
+.. code-block:: rest
+
+   Welcome to Lumache's documentation!
+   ===================================
+
+   **Lumache** (/luˈmake/) is a Python library for cooks and food lovers
+   that creates recipes mixing random ingredients.
+   It pulls data from the `Open Food Facts database <https://world.openfoodfacts.org/>`_
+   and offers a *simple* and *intuitive* API.
+
+   .. note::
+
+      This project is under active development.
+
+This showcases several features of the reStructuredText syntax, including:
+
+- a **section header** using ``===`` for the underline,
+- two examples of **inline markup**: ``**bold**`` and ``*italics*``,
```

Comment:
FYI: Accurately, these are `**strong emphasis**` and `*emphasis*`. Surely they're rendered as bold and italic usually. I don't know which wording is better for beginners. But we need to know they're different things and choose better representation.
https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#emphasis

refs: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/strong#%3Cb%3E_vs._%3Cstrong%3E

Reply:
You're right, let's use the original terminology here. Changing!